src/inc-lib-init.php

```diff
@@ -1084,336 +1084,488 @@ function lovd_getInstallURL ($bFull = true)
 
     static $aTranscriptOffsets = array();
     $aResponse = array(
         'position_start' => 0,
-        'position_end' => 0,
-        'type' => '',
-        'warnings' => array(),
+        'position_end'   => 0,
+        'type'           => '',
+        'warnings'       => array(),
+        'errors'         => array(),
     );
 
+    if (preg_match('/^[NX][MR]_[0-9]{6,9}\.[0-9]+:[cn]/', $sVariant)) {
+        if (is_numeric($sTranscriptID)) {
+            $sNCBIID = $_DB->query('SELECT id_ncbi FROM ' . TABLE_TRANSCRIPTS . ' WHERE id = ?',
+                array($sTranscriptID)) ->fetchColumn();
+        } else {
+            $sNCBIID = $sTranscriptID;
+        }
+
+        if ($sNCBIID == strstr($sVariant, ':', true)) {
+            $aResponse['warnings']['WTRANSCRIPTFOUND'] = 'Transcript ID found in the DNA description.';
+        } else {
+            $aResponse['warnings']['WDIFFERENTTRANSCRIPT'] =
+                'The transcript found in the DNA description does not match the configured transcript.';
+        }
+
+        $sVariant = substr(strstr($sVariant, ':'), 1);
+    }
+    
+    // All information of interest will be placed into an associative array.
+    // Notes: -All positions are filled by 0 if no position was matched.
+    //        -For now, the regular expression only works for c., g., n., and m. variants.
+    //        -If no information was matched for the variant type, parentheses and suffix,
+    //         these are filled by an empty string.
+    //        -Positions that were not given are cast to 0.
+    preg_match(
+        '/^([cngm])\.' .                  // 1.  Prefix
+
+        '([?=]$|(' .
+        '(\({1,2})?' .              // 4=(       // 4.  Opening parentheses
+        '([-*]?[0-9]+|\?)' .                     // 5.  (Earliest) start position
+        '([-+]([0-9]+|\?))?' .                   // 6.  (Earliest) intronic start position
+        '(?(4)(_' . 
+            '([-*]?[0-9]+|\?)' .                 // 9. Latest start position
+            '([-+]([0-9]+|\?))?' .               // 10. Latest intronic start position
+        '\))?)' .
+
+        '(_' .
+            '(\()?' .               // 13=(
+            '([-*]?[0-9]+|\?)' .                 // 14. (Earliest) end position
+            '([-+]([0-9]+|\?))?' .               // 15. (Earliest) intronic end position
+            '(?(13)_' . 
+                '([-*]?[0-9]+|\?)' .             // 17. Latest end position
+                '([-+]([0-9]+|\?))?' .           // 18. Latest intronic end position
+        '\)))?' .
+
+        '([ACGT]+>[ACGTRYSWKMBDHVN]+|' .         //  | (substitution)
+        '([ACTG]+\[[0-9]+])+' .                  //  | (repeat sequence)
+        '|ins|dup|delins|del|inv|[=?]' .         //  V
+        '|\|(gom|lom|met=))' .                   // 20. Type of variant
+
+        '(.*)))/',                               // 23. Suffix
+
+        $sVariant, $aMatches);
+
+    $aVariant = (!isset($aMatches[0]) ? [] : array(
```

Comment:
`[]` moet `array()` worden, aangezien dit PHP 5.4.0 vereist en we nog op 5.3.0 leunen.